ClemBot.Api/ClemBot.Api.Services/Caching/DesignatedChannels/DesignatedChannelsDetailsCacheHandler.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ClemBot.Api.Data.Contexts;
+using ClemBot.Api.Data.Models;
+using ClemBot.Api.Services.Caching.DesignatedChannels.Models;
+using LazyCache;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace ClemBot.Api.Services.Caching.DesignatedChannels;
+
+public class DesignatedChannelsCacheHandler: IRequestHandler<ClearDesignatedChannelDetailRequest>, IRequestHandler<GetDesignatedChannelDetailRequest, List<DesignatedChannelMapping>>
```

Comment:
this is super nitpicky but i can't believe it didn't catch this
```suggestion
public class DesignatedChannelsCacheHandler : IRequestHandler<ClearDesignatedChannelDetailRequest>, IRequestHandler<GetDesignatedChannelDetailRequest, List<DesignatedChannelMapping>>
```